README.rst

```diff
@@ -44,11 +43,15 @@ The following line should do the job::
 Quick Start
 -----------
 
-This graph, adapted from Figure 2 of the Metis
-`manual <http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/manual.pdf>`_ to
+This graph, adapted from Figure 2 of the METIS
+`manual <https://raw.githubusercontent.com/KarypisLab/METIS/master/manual/manual.pdf>`__ to
 use zero-based indexing,
 
-.. image:: doc/_static/tiny_01.png
+.. raw:: html
+
+    <div align="center">
+    <img src="https://raw.githubusercontent.com/inducer/pymetis/refs/heads/main/doc/_static/tiny_01.png" width=300><br>
+    </div>
```

Comment:
As far as I know, `.. image::` directives don't work on pypi (they're not actually shown [here](https://pypi.org/project/pymetis/)). This should work + they're nicely aligned :grin: